addon/templates/components/uni-anchor.hbs

```diff
@@ -1 +1,2 @@
 {{{label}}}
+{{yield}}
```

Comment:
You cannot simply replace label by yield. Everywhere we are using {{uni-anchor label="Label"}} will break. You got to support both.